### DISCUSSION TITLE
[IntelliSense] : add a license

### DISCUSSION THREAD

**Discussion by happysalada** [tailwindlabs/tailwindcss#12214] [Ideas]
Could you add a license to the repo ?
I couldn't find one.

**Comment by wongjn** [comment]
For tailwindlabs/tailwindcss-intellisense, they do have licenses for each of the packages that it contains, if that is useful to you:

- [tailwindcss-intellisense](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/packages/tailwindcss-language-server/LICENSE)
- [vscode-tailwindcss](https://github.com/tailwindlabs/tailwindcss-intellisense/tree/master/packages/vscode-tailwindcss/LICENSE)
- [tailwindcss-language-service](https://github.com/tailwindlabs/tailwindcss-intellisense/tree/master/packages/tailwindcss-language-service/LICENSE)

**Reply by happysalada** [reply]
Oh this can be closed then, thank you !
